components/bases-locales/bal-cover-map.js

```diff
@@ -112,6 +112,10 @@ class BalCoverMap extends React.Component {
 
     const [feature] = event.features
 
+    const _data = this.props.data.features.find(f =>
```

Comment:
Tu peux ajouter `data` à la déstructuration au dessus.
` const {map, data, popup} = this.props`